platform-configs/protectli-vp2430.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+    Resource    include/default.robot
+    Resource    include/protectli-vp24xx
+
+*** Variables ***
+
+*** Settings ***
+Resource    include/protectli-vp24xx.robot
+
+
+*** Variables ***
+${INITIAL_CPU_FREQUENCY}=           3300
+${FLASHING_METHOD}=                 external
+
+# eMMC driver support
+${E_MMC_NAME}=                      BJTD4R
+
+@{ATTACHED_USB}=                    ${TBD}
```

Comment:
Is this variable needed? It is only defined in `raptor-cs_talos2.robot` and set to @{EMPTY} in `protectli-vp2410`. The `usb-detect` test suite uses it and it seems unbelievable that this suite passess on any device apart from the raptor-cs_talos. Isn't the variable set automatically somewhere?

Reply:
Actually, even on the `raptor-cs_talos2` platform the variable is not set, because an undefined variable is set to it. It even is called to be temporary.
```
# Temporary parameter - we are not able to use the Hardware matrix
@{ATTACHED_USB}=                                    ${USB_LIVE}
```
I highly doubt the `@{ATTACHED_USB}` makes sense here. 
https://github.com/Dasharo/open-source-firmware-validation/pull/554/commits/0968c1602a5f3869625089d49a895037967e46c2

Reply:
There are open issues and PRs related to the usb detect test suite
https://github.com/Dasharo/open-source-firmware-validation/issues/279#issuecomment-2206059601
https://github.com/Dasharo/open-source-firmware-validation/pull/312
It is performed manually on every release for quite some time now, according to the test matrix, so I guess it's reasonable to just remove it until the test is fixed.